tests/CustomPost.php

```diff
@@ -8,4 +8,6 @@
 class CustomPost extends Model
 {
     protected $uuidField = 'customField';
```

Comment:
In the case where you have a UUID type defined in the casts array, why do you need the $uuidField attribute? Is it for when there are multiple UUIDs on the same model? Does this package support that use case?